doc/non-ansible-configuration-notes.md

```diff
@@ -130,63 +130,6 @@ Then, manually:
 * Update <https://github.com/nodejs/build/blob/master/ansible/inventory.yml> to reflect any host additions or changes (this is primarily for automatic .ssh/config setup purposes currently)
 * Run Ansible from <https://github.com/nodejs/build/tree/master/setup/raspberry-pi> with `ansible-playbook ansible-playbook.yaml -i ../ansible-inventory --limit <hostname>`
 
-## git on CentOS 5
```

Comment:
Maybe keep this for the time we are maintaining the machine manually.
Or even leave a "scar" pointing to this changeset (https://github.com/nodejs/build/pull/1454/files)

Reply:
@refack What's a "scar"?

Reply:
Metaphone for something that is left after a resection... Something like:
```
## CentOS 5
(removed in https://github.com/nodejs/build/pull/1454/files)
```